cmd/user-agent/scripts/setup_assets.sh

```diff
@@ -9,11 +9,9 @@ rm -rf public/aries-framework-go/
 mkdir -p public/aries-framework-go/assets
 npm install
 cp -Rp node_modules/@hyperledger/aries-framework-go/dist/assets/* public/aries-framework-go/assets
-gunzip public/aries-framework-go/assets/aries-js-worker.wasm.gz
```

Comment:
technically the module has a fallback to the original file (aries-js-worker.wasm)
https://github.com/lpar/gzipped/blob/master/fileserver.go#L129

I am guessing that if a browser doesn't send accept-encoding:zip, a 404 will now result.

Reply:
I'm not sure

Reply:
to support those clients:

```
cp public/aries-framework-go/assets/aries-js-worker.wasm.gz public/aries-framework-go/assets/aries-js-worker.wasm.gz.bak
gunzip public/aries-framework-go/assets/aries-js-worker.wasm.gz
mv public/aries-framework-go/assets/aries-js-worker.wasm.gz.bak public/aries-framework-go/assets/aries-js-worker.wasm.gz
```